Assets/_Main/Bruno/Scripts/RangedAI.cs

```diff
@@ -0,0 +1,122 @@
+using UnityEngine;
+using System.Collections.Generic;
+using System.Collections;
+
+public class RangedAI : Entity
+{
+    [SerializeField] private List<Transform> patrolPoints;

```

Comment:
Metti uno spazio tra questa variabile e quella successiva. Ricorda di dividere sempre i [SerializeField] dalle variabili normali. Guarda bene il document CodeStyleExample che si trova in Assets\_Main\_Common\Other